PR: https://github.com/odoo/odoo/pull/282884

From: 58c888ed3d7d57596ded39fc4e9337fc0ed4c7e2
From: svs-odoo
Date: 2026-08-28 20:53:36

Breaking data model changes scores: del:3 + add:0, change matches:
-    show_operations = fields.Boolean(related='picking_id.picking_type_id.show_operations')
-    show_operations = fields.Boolean(related='picking_type_id.show_operations')
-    show_operations = fields.Boolean(

Total Changes: 8

[REM] stock, *: remove dead code

*: mrp, mrp_subcontracting, purchase_stock, sale_stock

* Remove `show_operation`, this field is not used anymore and a old
  '#TODO' comment was asking to remove it since years;
* Remove `package_type_sequence_id` unused field;
* Remove `_compute_goods_received_not_invoiced` and
  `_compute_goods_delivered_not_invoiced`, those methods weren't used
  anymore since [^1]. In addition, their files were deleted too (in
  fact, they weren't even imported so it was kind of they were already
  deleted.)

[^1]: https://github.com/odoo/odoo/pull/247625/

Part-of: odoo/odoo#282884
Related: odoo/enterprise#128175
Related: odoo/upgrade#11054
Signed-off-by: William Henrotin (whe) <whe@odoo.com>

================================= pseudo patch: =================================

--- a/addons/stock/models/stock_move.py
+++ b/addons/stock/models/stock_move.py
@@ -176,8 +176,6 @@ class StockMove(models.Model):
     quantity_product_uom = fields.Float(
         'Quantity in Product UoM', digits='Product Unit',
         copy=False, compute='_compute_quantity_product_uom', store=True)
-    # TODO: delete this field `show_operations`
-    show_operations = fields.Boolean(related='picking_id.picking_type_id.show_operations')
     picking_code = fields.Selection(related='picking_type_id.code', readonly=True)
     show_details_visible = fields.Boolean('Details Visible', compute='_compute_show_details_visible')
     is_storable = fields.Boolean(related='product_id.is_storable')

--- a/addons/stock/models/stock_picking.py
+++ b/addons/stock/models/stock_picking.py
@@ -154,8 +154,6 @@ class StockPicking(models.Model):
     # Used to search on pickings
     product_id = fields.Many2one('product.product', 'Product', related='move_ids.product_id', readonly=True)
     lot_id = fields.Many2one('stock.lot', 'Lot/Serial Number', related='move_line_ids.lot_id', readonly=True)
-    # TODO: delete this field `show_operations`
-    show_operations = fields.Boolean(related='picking_type_id.show_operations')
     show_lots_text = fields.Boolean(compute='_compute_show_lots_text')
     has_tracking = fields.Boolean(compute='_compute_has_tracking')
     products_availability = fields.Char(

--- a/addons/stock/models/stock_picking_type.py
+++ b/addons/stock/models/stock_picking_type.py
@@ -62,10 +62,6 @@ class StockPickingType(models.Model):
     print_label = fields.Boolean(
         'Generate Shipping Labels', compute="_compute_print_label", store=True, readonly=False,
         help="Check this box if you want to generate shipping label in this operation.")
-    # TODO: delete this field `show_operations`
-    show_operations = fields.Boolean(
-        'Show Detailed Operations', default=False,
-        help="If this checkbox is ticked, the pickings lines will represent detailed stock operations. If not, the picking lines will represent an aggregate of detailed stock operations.")
     reservation_method = fields.Selection(
         [('at_confirm', 'At Confirmation'), ('manual', 'Manually'), ('by_date', 'Before scheduled date')],
         'Reservation Method', required=True, default='at_confirm',
